PureLayout/PureLayout/ALView+PureLayout.h

```diff
@@ -130,6 +146,7 @@ __PL_ASSUME_NONNULL_BEGIN
 /** Aligns an axis of the view to the same axis of another view with a multiplier. */
 - (NSLayoutConstraint *)autoAlignAxis:(ALAxis)axis toSameAxisOfView:(ALView *)otherView withMultiplier:(CGFloat)multiplier;
 
+- (__NSArray_of(NSLayoutConstraint *) *)autoAlignBothAxisToView:(ALView *)otherView;
```

Comment:
Similar to above this seems like `autoCenterInSuperview` but not for the immediate superview. I've never had to do this either...